changelog/_unreleased/2024-11-28-new-adresse-selection-modal.md

```diff
@@ -0,0 +1,30 @@
+---
+title: New adresse manager modal
+issue: NEXT-19776
+---
+# Core
+* Added `FEATURE_NEXT_19776` feature flag to enable/disable the new address manager modal and account address page redesign 
+___
+# Storefront
+* Deprecated `src/Storefront/Resources/views/storefront/component/address/address-editor-modal.html.twig`, use `src/Storefront/Resources/views/storefront/component/address/address-manager-modal.html.twig` instead
+* Deprecated `src/Storefront/Resources/views/storefront/component/address/address-editor-modal-list.html.twig`, use `src/Storefront/Resources/views/storefront/component/address/address-manager-modal-list.html.twig` instead
+* Deprecated `src/Storefront/Resources/views/storefront/component/address/address-editor-modal-create-address.html.twig`, use `src/Storefront/Resources/views/storefront/component/address/address-manager-modal-create-address.html.twig` instead
+* Changed `src/Storefront/Controller/AddressController.php` and added new routes `AddressController::addressManager`, `AddressController::addressManagerGet` and `AddressController::addressManagerSwitch` for the new address manager modal
```

Comment:
Additionally to "Core", we also have a headline for "API", where you could also state, what the route names are and what they are used for